plugins/woocommerce/src/Blocks/BlockTypes/MiniCart.php

```diff
@@ -367,7 +367,9 @@ protected function get_cart_price_markup( $attributes ) {
 		if ( isset( $attributes['hasHiddenPrice'] ) && false !== $attributes['hasHiddenPrice'] ) {
 			return;
 		}
-		$price_color = array_key_exists( 'priceColor', $attributes ) ? $attributes['priceColor']['color'] : '';
+		$price_color = isset( $attributes['priceColor'] ) && isset( $attributes['priceColor']['color'] )
+			? esc_attr( $attributes['priceColor']['color'] )
+			: '';
```

Comment:
It should be possible to use the null coalescing operator to simplify the check:
```suggestion
		$price_color = $attributes['priceColor']['color'] ?? '';
```
If you agree, please also apply to the `get_include_tax_label_markup()` function (line 388). Thanks!